style.css

```
@@ -0,0 +1,69 @@
+
+input:focus {outline:none;}
+* {
+    box-sizing: border-box;
+    font-family: 'SpoqaHanSansNeo-Regular';
```

폰트를 로드하지 못했을 때 기본 폰트를 제시해주시는 것도 좋을 것 같습니다!
```suggestion
    font-family: 'SpoqaHanSansNeo-Regular', sans-serif;
```